lib/domain/service/component_generator/component_generator_service.dart

```diff
@@ -86,15 +86,15 @@ class ComponentGeneratorService
     final declarationFilePath =
         sourceComponent.getDeclarationFilePath(projectLibFolder);
 
-    final declarationBody = sourceComponent.getDeclarationBody(projectName);
+    final declarationBody = sourceComponent.getSourceDeclarationBody(projectName);
```

Comment:
@cozvtieg9, changed the name from getDecralationBody to getSourceDeclarationBody to make it easier to understand where the surfaces are and where the repos are